src/dsm/headers/Street.hpp

```diff
@@ -31,15 +32,16 @@ namespace dsm {
   /// @tparam Size, The type of the street's capacity. It must be an unsigned integral type.
   class Street {
   private:
-    dsm::queue<Size> m_exitQueue;
+    std::vector<dsm::queue<Size>> m_exitQueues;
     std::set<Id> m_waitingAgents;
     std::pair<Id, Id> m_nodePair;
     double m_len;
     double m_maxSpeed;
     double m_angle;
     Id m_id;
     Size m_capacity;
-    Size m_transportCapacity;
+    int16_t m_transportCapacity;
+    int16_t m_nLanes;  // Max 255 lanes for a street, seems reasonable.
```

Comment:
Old comment or did you mean to use 8 bits?

Reply:
Old comment, gonna delete it